pandas/tests/util/test_assert_frame_equal.py

```diff
@@ -219,7 +219,7 @@ def test_frame_equal_unicode(df1, df2, msg, by_blocks_fixture, obj_fixture):
     #
     # Test ensures that `tm.assert_frame_equals` raises the right exception
     # when comparing DataFrames containing differing unicode objects.
-    msg = msg.format(obj=obj_fixture)
+    msg = f"{obj_fixture}"
```

Comment:
This is no longer testing for the complete message using the parametrised msg template?

Reply:
agreed, this looks incorrect